agent/src/containeroptions/containeroptionparser.cpp

```diff
@@ -44,6 +45,30 @@ void ContainerOptionParser::readConfigElement(const json_t *element)
     }
 
     m_options->setEnableWriteBuffer(wo);
+
+    if (wo == true)
+    {
+        bool enableTemporaryFilesystemWriteBuffer = false;
+        if(!JSONParser::read(element,
+                             "enableTemporaryFileSystemWriteBuffer",
+                             enableTemporaryFilesystemWriteBuffer)) {
+            std::string errorMessage("Could not parse config due to: 'enableTemporaryFilesystemWriteBuffer' not found.");
+            log_warn() << errorMessage;
```

Comment:
why don't we directly log_output the message like
log_warn() << "Could not parse config due to: 'enableTemporaryFilesystemWriteBuffer' not found.";